blockchain-beans/src/com/axonivy/ivy/process/element/blockchain/EthereumActivity.java

```diff
@@ -22,7 +22,7 @@
 import ch.ivyteam.ivy.scripting.exceptions.IvyScriptException;
 import ch.ivyteam.ivy.scripting.language.IIvyScriptContext;
 import ch.ivyteam.ivy.scripting.objects.CompositeObject;
-import ch.ivyteam.ivy.scripting.restricted.execution.IvyScriptCode;

```

Comment:
if you would use `var` you would not have such an import and the build never fails ;)

but fine for me!

Reply:
haha touché